data/cvr/samples/jetsons/jetsons_bedrock-precinct_cvr.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CastVoteRecordReport xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+ xsi:schemaLocation="http://itl.nist.gov/ns/voting/1500-103/v1 ../../../../../CastVoteRecords/NIST_V0_cast_vote_records.xsd"
+ xmlns="http://itl.nist.gov/ns/voting/1500-103/v1">
+    <CVR>
+        <BallotImage>
```

Comment:
See note on "Ballot Images" at the top of the review.

Reply:
removed all the <BallotImage/> elements